src/components/UserInfo.js

```diff
@@ -15,4 +16,7 @@ export default class UserInfo {
       this._nameElement.textContent = nameInput;
       this._jobElement.textContent = jobInput;
     }
-  }
\ No newline at end of file
+
+    setProfileAvatar(avatar) {
```

Comment:
It's great to have a separate method for updating the avatar, since it's controlled by a different form